Mathlib/NumberTheory/FunctionField.lean

```diff
@@ -244,11 +244,11 @@ def inftyValuedFqt : Valued (RatFunc Fq) ℤₘ₀ :=
 set_option linter.uppercaseLean3 false in
 #align function_field.infty_valued_Fqt FunctionField.inftyValuedFqt
 
-theorem inftyValuedFqt.def {x : RatFunc Fq} :
+theorem inftyValuedFqt.def' {x : RatFunc Fq} :
```

Comment:
```suggestion
-- Adaptation note: 2024-03-15
-- Renamed to avoid the reserved name `inftyValuedFqt.def`.
theorem inftyValuedFqt.def' {x : RatFunc Fq} :
```